### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by John-Nagle** [schell/renderling#139] [Q&A] [answered]
Built on Ubuntu 22.04 LTS. Passes "cargo test".

example -m cube.glb

shows the cube as all black against grey. The other examples are all unlit, too. The emissive ones show some light.
How do I get lighting?

A white rectangle at the upper left shows INFFPS, then the text disappears.

example-culling panics at an unwrap at line 310 of main.rs, "Surface { source: Outdated }"

**Comment by Firestar99** [comment, marked as answer]
This works for me, run from project root:
`cargo run --bin example --release -- -m ./gltf/DamagedHelmet.glb -s ./img/hdr/resting_place.hdr`

You need to specify a `--skybox` (`-s`) for you to actually have something that emits light, as unfortunately, the default skybox is just black. The text is also broken for me on Ubuntu, though I never got it to panic. I only know this cause I randomly decided to try it out a few weeks ago.

**Reply by John-Nagle** [reply]
Ah. Specified a skybox, and now I can look at both the test scenes and Kronos test scenes. Looks nice.

**Comment by schell** [comment]
Thanks for the report @John-Nagle! 

Yes, the gltf viewer doesn't include analytical lighting at the moment (things have been changing quickly, recently). Including an HDR image for the skybox should suffice.

The text issue I'm pretty sure I've fixed on a branch, but I have a couple of those in the air at the moment. Unfortunately I don't have a linux box to test on outside of CI. I'll try to recreate your issue in a unit test, though.
